src/commands/run.rs

```diff
@@ -287,7 +288,7 @@ async fn build_store_instance(
                 }
             }
             _ => {
-                bail!("invalid url: currently slight only supports 'configs.usersecrets', 'configs.envvars', 'keyvalue.filesystem', 'keyvalue.azblob', 'keyvalue.awsdynamodb', 'distributed_locking.etcd', 'messaging.confluent_apache_kafka', 'messaging.mosquitto', and 'http' schemes")
+                bail!("invalid url: currently slight only supports 'configs.usersecrets', 'configs.envvars', 'keyvalue.filesystem', 'keyvalue.azblob', 'keyvalue.awsdynamodb', 'distributed_locking.etcd', 'messaging.nats', 'messaging.confluent_apache_kafka', 'messaging.mosquitto', and 'http' schemes")
```

Comment:
It would be nice to generate this from the const `MESSAGING_HOST_IMPLEMENTORS`. It's error prone to add a string to one place and need to add another to the message.

Reply:
That's fair~ I'll make that in another PR.

Reply:
~see #357